controller/cache/admission.go

```diff
@@ -1397,10 +1405,24 @@ func isAdmissionRuleMet(admResObject *nvsysadmission.AdmResObject, c *nvsysadmis
 			met, positive = isSetCriterionMet(crt, c.ImageRegistry)
 		//case share.CriteriaKeyBaseImage: //-> is base image available?
 		//	met, positive = isStringCriterionMet(crt, scannedImage.BaseName)
+		case share.CriteriaKeyCVECriticalCount:
+			met, positive = isCveCountCriterionMet(crt, false, 0, scannedImage.CriticalVulInfo)
+		case share.CriteriaKeyCVEHighCountNoCritical:
+			met, positive = isCveCountCriterionMet(crt, false, 0, scannedImage.HighVulInfo)
 		case share.CriteriaKeyCVEHighCount:
 			met, positive = isCveCountCriterionMet(crt, false, 0, scannedImage.HighVulInfo)
+			if !met {
+				met, positive = isCveCountCriterionMet(crt, false, 0, scannedImage.CriticalVulInfo)
+			}
 		case share.CriteriaKeyCVEMediumCount:
 			met, positive = isCveCountCriterionMet(crt, false, 0, scannedImage.MediumVulInfo)
+		case share.CriteriaKeyCVECriticalWithFixCount:
+			met, positive = isCveCountCriterionMet(crt, true, scannedImage.CriticalVulsWithFix, scannedImage.CriticalVulInfo)
+		case share.CriteriaKeyCVEHighWithFixCountNoCritical:
+			met, positive = isCveCountCriterionMet(crt, true, scannedImage.HighVulsWithFix, scannedImage.HighVulInfo)
+			if !met {
+				met, positive = isCveCountCriterionMet(crt, true, scannedImage.CriticalVulsWithFix, scannedImage.CriticalVulInfo)
+			}
 		case share.CriteriaKeyCVEHighWithFixCount:
 			met, positive = isCveCountCriterionMet(crt, true, scannedImage.HighVulsWithFix, scannedImage.HighVulInfo)
```

Comment:
share.CriteriaKeyCVEHighCount is "more than %s high and critical severity CVEs that were reported before %s days ago"
Why scannedImage.CriticalVulsWithFix is not checked in case share.CriteriaKeyCVEHighWithFixCount ?